File: rest/sip-in/create-ip-acl/create-ip-acl.5.x.cs
// Download the twilio-csharp library from
// https://www.twilio.com/docs/libraries/csharp#installation
using System;
using Twilio;
using Twilio.Rest.Trunking.V1.Trunk;

class Example
{
    static void Main(string[] args)
    {
        // Find your Account Sid and Auth Token at twilio.com/user/account
        string accountSid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
        string authToken = "your_auth_token";

        TwilioClient.Init(accountSid, authToken);

        var ipAccessControlList = IpAccessControlListResource.Create(
            "My new acl", "CLXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX");

        Console.WriteLine(ipAccessControlList.Sid);
    }
}

File: rest/sip-in/create-ip-acl/create-ip-acl.5.x.rb
require 'rubygems' # not necessary with ruby 1.9 but included for completeness
require 'twilio-ruby'

# put your own credentials here
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'

# set up a client to talk to the Twilio REST API
@client = Twilio::REST::Client.new(account_sid, auth_token)

ip_access_control_list = @client.sip.ip_access_control_lists
                                .create(friendly_name: 'My new acl')

puts ip_access_control_list.friendly_name
